src/view/dataElements/xdmObject.jsx

```diff
@@ -47,13 +45,6 @@ const XdmObject = ({ initInfo, formikProps, schema }) => {
     );
   }
 
-  const { formStateNode, fieldName, breadcrumb } = selectedNodeId
-    ? getNodeEditData({
-        formState,
-        nodeId: selectedNodeId
-      })
-    : {};
-
```

Comment:
This stuff and the breadcrumb below were specific to the node edit view, so they've been moved there.